charts/opentelemetry-kube-stack/templates/_helpers.tpl

```diff
@@ -121,6 +121,7 @@ helm.sh/chart: {{ include "opentelemetry-kube-stack.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+release: {{ $.Release.Name | quote }}
```

Comment:
in this case it should be `.Release.Name` as this is down in a helper.

Reply:
I fixed it and regenerated the examples as you suggested. Thank you!!

Reply:
thank you! running tests now, will merge upon success.